File: Python/model/denseMatrix.py
class DenseMatrix:

    def __init__(self, matrix, N):
        self.matrix = matrix
        self.N = N
    
    def display_matrix(self):
        for i in range(self.N):
            for j in range(self.N):
                print(self.matrix[i][j], end=' ')
            print()
        print()
        

File: src/main/java/mapreduce/matrix/DenseMatrix.java
package mapreduce.matrix;
import mapreduce.Matrix;

public class DenseMatrix implements Matrix {
    private final double[][] values;

    public DenseMatrix(double[][] values) {
        this.values = values;
    }

    @Override
    public int size() {
        return values.length;
    }

    @Override
    public double get(int i, int j) {
        return values[i][j];
    }
}